Plots/Trajectory/README.rst

```diff
@@ -0,0 +1,6 @@
+.. _trajectory_examples:
```

Comment:
This file as well as "Trajectory" folder should be removed because the correct folder is "Trajectories".